src/test/groovy/unit/org/pih/warehouse/api/ProductApiTest.groovy

```diff
@@ -0,0 +1,178 @@
+package unit.org.pih.warehouse.api
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import groovy.transform.builder.Builder
+import io.restassured.RestAssured
+import io.restassured.builder.RequestSpecBuilder
+import io.restassured.http.ContentType
+import io.restassured.http.Cookie
+import io.restassured.path.json.JsonPath
+import io.restassured.specification.RequestSpecification
+import lombok.AllArgsConstructor
+import lombok.Data
+import lombok.NoArgsConstructor
+import org.grails.web.json.JSONObject
+import spock.lang.Shared
+import spock.lang.Specification
+import static io.restassured.RestAssured.*
```

Comment:
the `*` imports have not been fixed (I've mentioned that in my initial review)

Reply:
I wish we had something like pmd checkstyle (I don't know if it is available in Groovy/Grails), but it would catch those things.
I also think that the imports should be grouped differently - the static imports should be at the beginning and be separated with one empty line, but since we don't have any strict rules regarding that, you can skip that grouping - just wanted to mention that.

Reply:
Agreed, it would be nice.